.pre-commit-config.yaml

```diff
@@ -1,23 +1,47 @@
 repos:
-  - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v3.4.0
+  - repo: local
```

Comment:
This way pre-commit runs the same versions that we (developers) run locally. All versioning is done by poetry.